docs/src/howto/async-commit-merge.md

```diff
@@ -0,0 +1,436 @@
+    "message": "Operation expired"
+  },
+  "status_code": 408
+}
+```
+
+Resolution: The operation may have completed but wasn't polled in time, or the operation legitimately timed out. For very large operations, consider breaking them into smaller chunks.
+
+**Not Implemented (501):**
+
+When attempting to use async endpoints on lakeFS OSS, you'll receive a 501 response from the initial POST request (not from the status endpoint).
+
+Resolution: You're using lakeFS OSS. Upgrade to lakeFS Enterprise or use the synchronous commit/merge endpoints.
+
+## Best Practices
+
+### 1. Always Persist Operation IDs
+
+Store operation IDs in your application state, database, or CI/CD pipeline variables so you can query status even if your process restarts. For example, you could store them in Redis, a database, or environment variables in your CI/CD system.
+
+### 2. Implement Timeout Handling
+
+Set your own application-level timeout if you need faster failure detection. Track the elapsed time since submission and exit the polling loop if your timeout threshold is exceeded.
+
+### 3. Use Appropriate Polling Intervals
+
+Balance between responsiveness and server load:
+
+- **Quick operations** (<1 min expected): Poll every 1-2 seconds initially
+- **Medium operations** (1-5 min expected): Start at 2s, back off to 10s
+- **Large operations** (>5 min expected): Start at 5s, back off to 30s
+
+### 4. Handle Network Failures Gracefully
+
+Network interruptions don't affect the async operation itself. Implement retry logic with exponential backoff when polling the status endpoint to handle transient network issues.
+
+### 5. Log Operation IDs
+
+Include operation IDs in your application logs for troubleshooting. Log when operations are submitted and when they complete to help track down issues in production.
+
+## Permissions
+
+Async operations require the same RBAC permissions as their synchronous counterparts. See [Commit and Merge documentation](../../quickstart/commit-and-merge/) for the full list of required permissions.
+
+The status endpoints also require these same permissions, as they check authorization against the original operation's repository and branch.
+
+## Limitations
+
+- **No cancellation**: Once submitted, async operations cannot be canceled. They will run to completion or timeout.
+- **Enterprise only**: Async operations are not available in lakeFS OSS.
+- **No SDK wrapper yet**: High-level SDK support is planned but not yet available. Use the REST API directly for now.
+- **Concurrent operations**: Multiple operations can be submitted to the same branch and will run in parallel. They coordinate through branch locking, which may cause temporary lock errors under heavy contention. See [Branch Lock Contention](#branch-lock-contention) in Troubleshooting for details.
+
+## Troubleshooting
+
+### Operation Not Found
+
+**Symptom:** Status endpoint returns 404 for a valid operation ID.
+
+**Possible causes:**
+- Operation ID was mistyped
```

Comment:
```suggestion
**Possible causes:**

- Operation ID was mistyped
```

Reply:
This probably needs the extra empty line to render properly. 

Currently it looks like this:

<img width="1039" height="328" alt="Image" src="https://github.com/user-attachments/assets/3396b564-be9a-40a6-8571-bd0b720e2227" />

Reply:
Same is true for the other lists in this section.